nullaway/src/main/java/com/uber/nullaway/handlers/Handlers.java

```diff
@@ -46,6 +46,7 @@ public static Handler buildDefault(Config config) {
     if (config.isJarInferEnabled()) {
       handlerListBuilder.add(new InferredJARModelsHandler(config));
     }
+    handlerListBuilder.add(new AssertionHandler());
```

Comment:
Let's gate this with a config flag, since we mostly just want to run it on test code.

Reply:
Yes, we should definitely gate with a config flag to avoid additional overhead.  It'll be off by default and folks who want it on their test code can enable.

Reply:
Added a config flag which is turned off by default.